docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,17 +1,17 @@
 ---
 title: Teleport Trusted Clusters
-description: How to configure access and trust between two SSH and Kubernetes environments.
+description: How to configure access and trust between two Teleport Clusters.
 h1: Trusted Clusters
 ---
 
 Teleport can partition compute infrastructure into multiple clusters. A cluster
-is a group of Teleport resources connected. Each cluster
+is a group of resources connected to a Teleport Auth Server. Each cluster
 manages a separate certificate authority (CA) for all users and resources.
 
 Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly SSH into the Nodes of another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-be running behind a firewall without any ingress ports open.
+seamlessly connect to resources in another cluster, the **leaf cluster**, while
+remaining authenticated with only a single Auth Service. The leaf cluster can be
+running behind a firewall without any ingress ports open.
```

Comment:
Removing 'be' phrasing. Ref: http://writing.rocks/to-be-or-not-to-be/

```suggestion
remaining authenticated with only a single Auth Service. The leaf cluster can run
behind a firewall without any ingress ports open.
```

Reply:
Great tip, thanks for sharing!